internal/step/spinner.go

```diff
@@ -75,11 +76,11 @@ func (s *stepWithSpinner) LogInfof(format string, args ...interface{}) {
 }
 
 func (s *stepWithSpinner) LogError(msg string) {
-	s.logTof(os.Stderr, warningGlyph+msg)
+	s.logTof(os.Stderr, color.YellowString(warningGlyph)+msg)
```

Comment:
Does windows terminal correctly display colorful output?

Reply:
No, that's why I moved it into the non-windows code base